APIs/schemas/sender.json

```diff
@@ -61,13 +61,13 @@
           }
         },
         "subscription": {
-          "description": "Object containing the 'receiver_id' currently subscribed to (unicast only).",
+          "description": "Object indicating how this Sender is currently configured to send data.",
           "type": "object",
           "required": ["receiver_id", "active"],
           "properties": {
             "receiver_id": {
               "type": ["string", "null"],
-              "description": "UUID of the Receiver that this Sender is currently subscribed to",
+              "description": "UUID of the Receiver to which this Sender is currently configured to send data, if it is active, uses a unicast push-based transport and is sending to an NMOS Receiver, otherwise null",
```

Comment:
```suggestion
              "description": "UUID of the Receiver to which this Sender is currently configured to send data. Only set if it is active, uses a unicast push-based transport and is sending to an NMOS Receiver; otherwise null.",
```